build.gradle

```diff
@@ -17,13 +17,16 @@ android {
     compileSdkVersion 21
```

Comment:
@petter-fidesmo lint complains because the current buildToolsVersion is 22.2.0.
Actually I had to install 21.1 so I could build this repo, so I guess it's been 22.2 for a while now.